PostHog/Classes/PHGPostHog.m

```diff
@@ -312,6 +311,71 @@ - (bool)isFeatureEnabled:(NSString *)flagKey
     return isFlagEnabled;
 }
 
+- (NSString *)getFeatureFlagStringPayload:(NSString *)flagKey defaultValue:(NSString *)defaultValue
+{
+    NSDictionary *payloads = [self.payloadManager getFeatureFlagPayloads];
+    NSString *payload = [payloads valueForKey:flagKey];
+    
+    if ([payload isKindOfClass:[NSString class]]) {
+        return payload;
+    } else {
+        NSLog(@"Could not retrieve value of type: NSString");
```

Comment:
I think we might want to prefix this with `[PostHog]` or something.

Reply:
Also, asking because I haven't followed the overall upgrade much, will there always be a payload? If the person isn't in capture group then they will likely have `nil` here right? Then we log an issue when there isn't really an issue 🤔

Reply:
Good catch!